articles-aem-renderer/src/androidTest/java/org/godtools/articles_aem_renderer/ExampleInstrumentedTest.java

```diff
@@ -0,0 +1,26 @@
+package org.godtools.articles_aem_renderer;
+
+import android.content.Context;
+import android.support.test.InstrumentationRegistry;
+import android.support.test.runner.AndroidJUnit4;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import static org.junit.Assert.*;
+
+/**
+ * Instrumented test, which will execute on an Android device.
+ *
+ * @see <a href="http://d.android.com/tools/testing">Testing documentation</a>
+ */
+@RunWith(AndroidJUnit4.class)
+public class ExampleInstrumentedTest {
```

Comment:
this test can be removed, it doesn't do anything, guessing it was an autogenerated test

Reply:
will do.